### ISSUE TITLE
Improve the model selector and select menu

### ISSUE THREAD

**Issue by Misjohns** [instructlab/ui#595]
### Goal
Improve the styling of the model selection menus to aid the user in selecting the appropriate model.
- Provide some visual affordance to help to distinguish between different model types/stages
– Base/Foundational (Implemented/Deployed)
– Fine-tuned/Customized (Post SDG)
– Pre-trained/Seed (RAG only, Chat with documents, Actively contributing, Data collection phase)
- Provide timestamps of when a model was created/modified
- Show recent category in menu list that shows last top 5 models accessed
- Include ability to filter the models menu

### Definition of Done
* Review and validate requirements
* Create wireframe/lo-fi designs
* Brainstorm & collaborate during IxD meetings
* Review with stakeholders at PM sync
* Share during UI community meeting
* Share on the appropriate Slack/Discord channels
* Resolve all comments in the GitHub
* Participate in grooming with Eng team

### Open items
- We need to settle to on the right terms for the different model types/stages
- Icons to represent the model types/stages
-- Utilizing FPO icons in mock-ups
-- RH branding to produce final resources

### Related issues
Improve design of Chat with Model #438
Improve design of Custom Model Endpoint #460   

### Resources
[Figma](https://www.figma.com/design/eOCwKx5HH3SvaH0NtR1U79/MVP-RHELAI---Contributor-flow?node-id=1060-24114&t=T6HzlixkvPXcfCeJ-4)
[UXD Presentation](https://docs.google.com/presentation/d/1khcwNr25dT1Agun3PyKpHfoJVX4EvJZM8Ti0yU-MvLg/edit?usp=sharing)

**Comment by Misjohns** [comment]
### Model selector default state
Model item includes:
- Model icon
- Model name
- Model type (Base or Fine-tuned)

![Image](https://github.com/user-attachments/assets/a8ae7fdb-4210-478c-963f-5fa48362ec20)

### Model menu list
Menu groups:
- Recent (Last 5)
- Fine-tuned
- Base
- Model list items
- Menu action (Manage models)
-- This link could navigate the user to a page or modal where user could add/remove access to models

Model list item includes:
- Model icon
- Model name
- Model type (Base or Fine-tuned)
- Timestamp (Created or Updated)
- Selected option mark (blue check)

![Image](https://github.com/user-attachments/assets/8e8f71b5-ca2d-4c74-949c-061372c0c00b)

**Comment by Misjohns** [comment]
@beaumorley @Mdenisco @vishnoianil @nerdalert Please review the design and verify the requirements.

**Comment by Misjohns** [comment]
@jehlum11 @beaumorley @mairin @nerdalert @vishnoianil @williamcaban
We need to finalize what terminology and labels we use to distinguish between the 2 types/stages of models in InstructLab. In the mockups I am using **Fine-tuned** and **Base** but those are placeholders based on conversations we had back in 2024.

- [IBM LAB method](https://miro.com/app/board/uXjVIcwVzKg=/?moveToWidget=3458764618807450252&cot=14) uses the terms foundational and teacher model. 
- Looks like RHOAI has recommendations for LAB tune, LAB starter, LAB teacher, LAB judge. Not sure which stage in the process these labels are referring to, but I did want to reference those discussions so we can align across products.

Once finalized, I can update these mockups and then pass this issue on to dev.
